pyproject.toml

```diff
@@ -49,12 +49,12 @@ smolagents = [
 ]
 
 a2a = [
-  "a2a-sdk>=0.2.11",
+  "a2a-sdk>=0.3.0",
   "httpx",
 ]
 
 all = [
-  "any-agent[agno,google,langchain,llama_index,openai,smolagents]",
+  "any-agent[a2a,agno,google,langchain,llama_index,openai,smolagents]",
```

Comment:
any reason for this change? 

As far as I see, the any-agent sample in a2a installs the correct `a2a` extra:

https://github.com/a2aproject/a2a-samples/blob/main/samples/python/agents/any_agent_adversarial_multiagent/pyproject.toml#L8